pkg/extensions/search/search.md

```diff
@@ -1,186 +1,673 @@
-# `search` 
+# `search`
 
 `search` component provides efficient and enhanced registry search capabilities using graphQL backend.
-	
+
 | Supported queries | Input | Ouput | Description | graphQL query |
 | --- | --- | --- | --- | --- |
-| [Search images by digest](#search-images-by-digest) | digest | image list | Search all repositories in the registry and return list of images that matches given digest (manifest, config or layers) | ImageListForDigest |
```

Comment:
Why change the position of these? Would ideally either amend-in-place or append new entries

Reply:
Will update the doc. No reason that I am aware of.